### ISSUE TITLE
bug: Popover not positioned correct with targetDim.left = 0

### ISSUE THREAD

**Issue by flavordaaave** [driftyco/ionic#6896]
**Type**: <span ionic-type>bug</span>

**Ionic Version**: <span ionic-version>2.x</span>

**Platform**: <span ionic-platform>ios</span> <span ionic-platform-version>9</span>  <span ionic-webview>browser</span>

<span ionic-description>Steps to reproduce:
1) Create a div that is positioned with left: 0 within the viewport
2) Add a (click) handler on the div to open a popover
3) The popover is not positioned according to the div but in the middle of the viewport

I check the source and I think this is because of this line (see: https://github.com/driftyco/ionic/blob/2.0/src/components/popover/popover.ts#L332)
```
let targetLeft = targetDim && targetDim.left || bodyWidth / 2;
```

Since targetDim.left = 0 it equals `false` and will fallback to `bodyWidth / 2` which you can also see in the console.logs in the attached screenshot (I clicked on the yellow highlighted div).</span>

![popover_misaligned](https://cloud.githubusercontent.com/assets/5305150/16040856/9925b120-3232-11e6-9aff-6b7bb8551cdc.png)

<span is-issue-template></span>

**Comment by flavordaaave** [comment]
Not sure if this is the best solution but for me changing the lines 331 and 332 to this worked:
```
let targetTop = (targetDim && 'top' in targetDim)  ? targetDim.top : (bodyHeight / 2) - (popoverHeight / 2);
let targetLeft = (targetDim && 'left' in targetDim) ? targetDim.left : bodyWidth / 2;
```

**Comment by brandyscarney** [comment]
Fixed by PR #6900 by @flavordaaave. Thanks!
